OrbitCore/CaptureData.h

```diff
@@ -27,7 +28,8 @@ class CaptureData {
         selected_functions_{std::move(selected_functions)},
         selected_tracepoints_{std::move(selected_tracepoints)},
         callstack_data_(std::make_unique<CallstackData>()),
-        selection_callstack_data_(std::make_unique<CallstackData>()) {
+        selection_callstack_data_(std::make_unique<CallstackData>()),
+        tracepoint_info_manager_(std::make_unique<TracepointInfoManager>()) {
```

Comment:
`TracepointEventBuffer` was the one supposed to end up in `CaptureData`, not `TracepointInfoManager` (or both, but `TracepointEventBuffer` was the object of the discussion).